experiments/AMIP/moist_mpi_earth/slab_ocean/slab_init.jl

```diff
@@ -37,11 +37,11 @@ function slab_ocean_space_init(::Type{FT}, space, p) where {FT}
 end
 
 # ode
-function slab_ocean_rhs!(dY, Y, Ya, t)
-    p, F_aero, F_rad, land_mask = Ya
+function slab_ocean_rhs!(dY, Y, cache, t)
+    p, F_aero, F_rad, land_mask = cache
     FT = eltype(Y.T_sfc)
     rhs = @. -(F_aero + F_rad) / (p.h * p.ρ * p.c)
-    parent(dY.T_sfc) .= apply_mask.(FT, parent(land_mask), <, parent(rhs))
+    parent(dY.T_sfc) .= apply_mask.(FT, parent(land_mask), parent(rhs))
```

Comment:
would it be clearer to use the ocean_mask here?

Reply:
in the conservation checker, we multiply the e.g. ocean field by ocean_mask. Is it possible to do that here (do we still need `apply_mask`?

Reply:
(similar question for the other slab RHS)

Reply:
It's doing a slightly different thing here. In the conservation checker it's doing the weighted sum (so near the domain boundaries the ocean temperature is `T_sfc_ocean / ocean_fraction`), whereas here the `apply_mask` cuts out the ocean domain area, with `T_sfc_ocean` retaining the absolute value (i.e. not dividing by the `ocean_fraction`)

Reply:
But I have changed the above to the ocean mask, which makes more sense. :)